src/components/thumbnail/documents-type-collection.tsx

```diff
@@ -90,22 +94,24 @@ export const DocumentCollectionByType = observer(({ tab, section, index, numSect
     const showNewDocumentThumbnail = section.addDocument && !!onSelectNewDocument;
     const newDocumentLabel = getNewDocumentLabel(section, appConfigStore);
     const isTopPanel = index === 0 && numSections > 1;
-    const isBottomPanel = index === numSections - 1 && index > 0;
+    const isBottomPanel = index > 0;
```

Comment:
Wouldn't this make every panel after the first one the bottom panel? Are we assuming we only have two panels?